### PR TITLE
Fix changed-files

### PR DESCRIPTION
**Pull request by poshul**
Path spec should be part of `files:` not `path:`

### CODE REVIEW

**Comment by jpfeuffer** [comment]
Is this going to be recursive? I want to have the option to have subfolders under source. Maybe there are some currently already.

**Comment by poshul** [comment]
> Is this going to be recursive? I want to have the option to have subfolders under source. Maybe there are some currently already.

We have couple of things in `docs/source/_templates` I'll make a branch and see what the best way to handle this is.

File: .github/workflows/test-pr.yaml [hunk 1]
```diff
@@ -24,6 +24,7 @@ jobs:
           dir_names_max_depth: 0
           files: |
             docs/source/*.rst
+            docs/source/**/*.rst
```
Comment (jpfeuffer) [inline .github/workflows/test-pr.yaml:27]: this looks like it will still check the _templates, right?

**Comment by poshul** [comment]
Yup.

**Comment by jpfeuffer** [comment]
Ah ok. I thought you were looking to exclude templates :D We actually don't need to check them.

Can you do the same for the other actions? For rst to ipynb conversion we actually NEED to exclude _templates.

**Comment by poshul** [comment]
Ahh, I parsed your response wrong then. It looks like there is a possibility to exclude directories with `!`. I can add that as a line here as well.

**Review by jpfeuffer — APPROVED** [review]

**Pull request merged**